searx/search/processors/online_dictionary.py

```diff
@@ -9,7 +9,7 @@
 from searx.utils import is_valid_lang
 from .online import OnlineProcessor
 
-parser_re = re.compile('.*?([a-z]+)-([a-z]+) ([^ ]+)$', re.I)
+parser_re = re.compile('.*?([a-z]+)-([a-z]+) (.*)$', re.I)
```

Comment:
just curious, why not use `.*?([a-z]+)-([a-z]+) (.+)$` instead?

Reply:
Not all languages use the latin alphabet.
Even in English that's not so easy, for example `o'clock`

Reply:
sorry, but what do you mean? I only edited the `(.*)` to `(.+)` and it seems to work...

Reply:
> sorry, but what do you mean?

I've read too quickly, sorry. 

I agree with `(.+)`, it avoids empty queries.